include/quill/backend/BackendUtilities.h

```diff
@@ -84,12 +85,26 @@ QUILL_ATTRIBUTE_COLD inline void set_cpu_affinity(uint16_t cpu_id)
   thread_port_t mach_thread = pthread_mach_thread_np(pthread_self());
 
   thread_policy_set(mach_thread, THREAD_AFFINITY_POLICY, (thread_policy_t)&policy, 1);
+#else
```

Comment:
Looks like this is also `elif` ?

Reply:
I wanted to reuse the same error checking bellow on l.109 so that's why a nested `#if`.
Initial I had `elif` but this caused build failures on mac and win because of missing `err`.